taskcluster/ci/browsertime/kind.yml

```diff
@@ -244,3 +244,44 @@ jobs:
                 armeabi-v7a:
                     - '--test-url-params=exclude=1,2,9,10,17,18,21,22,26,28,30,32,39,40,47,48,55,56,63,64,71,72,79,80,83,84,89,90,95,96'
                 default: []
+
+    youtube-playback-av1-sfr:
```

Comment:
We should make some changes in the transform to only run this AV1 test on pixel 2.